releng/setup-env.sh

```diff
@@ -267,7 +267,6 @@ case $host_platform in
     libstdcxx_flags="-static-libstdc++"
     base_compiler_flags="-ffunction-sections -fdata-sections"
     base_linker_flags="-Wl,--gc-sections -Wl,-z,noexecstack $libgcc_flags"
-
```

Comment:
This line should be kept. It's there to visually separate the groups of variables.